app/uploaders/user_icon_uploader.rb

```diff
@@ -0,0 +1,51 @@
+class UserIconUploader < CarrierWave::Uploader::Base
+  # Include RMagick or MiniMagick support:
+  # include CarrierWave::RMagick
+  include CarrierWave::MiniMagick
+
+  # Choose what kind of storage to use for this uploader:
+  # storage :file
+  # storage :fog
+
+  # Override the directory where uploaded files will be stored.
+  # This is a sensible default for uploaders that are meant to be mounted:
+  def store_dir
+    if Rails.env.test?
+        Rails.root.join("spec", "support", "uploads", model.class.to_s.underscore, mounted_as, model.id)
+      else
+        "uploads/#{model.class.to_s.underscore}/#{mounted_as}/#{model.id}"
+      end
```

Comment:
__[RuboCop]__

Lint/EndAlignment: `end` at 17, 6 is not aligned with `if` at 13, 4.


<!-- Comment by [SideCI](https://sideci.com) -->